### PR TITLE
feat(behavior_path_planner): update turn signal document

### PR DESCRIPTION
**Pull request by purewater0901**
## Description

Add parameter descriptions to the turn signal decider document. 

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by kosuke55** [comment]
Please update 3.0 -> turn_signal_search_time also for pull_over

image

**Review by rej55 — COMMENTED** [review]
Thank you for contribution.
Default value in the document seems different from config file.
Could you check it?

File: planning/behavior_path_planner/behavior_path_planner_turn_signal-design.md [hunk 1]
```diff
@@ -6,10 +6,30 @@ Turn Signal decider determines necessary blinkers.
...
 
+## Parameters for turn signal decider
+
+| Name                                            | Unit | Type   | Description                                                                  | Default value |
+| :---------------------------------------------- | :--- | :----- | :--------------------------------------------------------------------------- | :------------ |
+| turn_signal_intersection_search_distance        | [m]  | double | constant search distance to decide activation of blinkers at intersections   | 30            |
+| turn_signal_intersection_angle_threshold_degree | deg  | double | angle threshold to determined the end point of intersection required section | 15            |
+| turn_signal_minimum_search_distance             | [m]  | double | minimum search distance for avoidance and lane change                        | 10            |
+| turn_signal_search_time                         | [s]  | double | search time for to decide activation of blinkers                             | 10            |
```
[the remark this answers is not in the document]
Comment (purewater0901) [reply planning/behavior_path_planner/behavior_path_planner_turn_signal-design.md]: done

[the remark this answers is not in the document]
Comment (purewater0901) [reply planning/behavior_path_planner/behavior_path_planner_turn_signal-design.md]: @rej55 Sorry, it was my mistake. I just fixed it.

**Review by rej55 — APPROVED** [review]

**Pull request merged**